cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtilsTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet.parameters;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import org.easymock.EasyMock;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class ParameterUtilsTest {
+
+  private static final Long DEFAULT_START_TIME_MS = -1L;
+  private static final Long DEFAULT_END_TIME_MS = -2L;
+
+  @Test
+  public void testParseTimeRangeSet() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    String startTimeString = "12345";
+    String endTimeString = "23456";
```

Comment:
Can we avoid hardcoding test values and move them to static variables?

Reply:
Sure